src/webgl/framebuffer.js

```diff
@@ -388,14 +453,16 @@ export default class Framebuffer extends Resource {
   // pname returned type
   // If an OpenGL error is generated, returns null.
   getAttachmentParameter({
-    target = this.target,
     attachment = GL_COLOR_ATTACHMENT0,
     pname
   } = {}) {
-    const fallback = this._getAttachmentParameterFallback(pname);
-    return fallback !== null ?
-      fallback :
-      this.gl.getFramebufferAttachmentParameter(target, attachment, pname);
+    let value = this._getAttachmentParameterFallback(pname);
+    if (value === null) {
+      this.gl.bindTexture(GL_FRAMEBUFFER, this.handle);
```

Comment:
Nit: update comments
`passed pname given the target and attachment.` -> `passed pname of given attachment for 'FRAMEBUFFER' target`

Reply:
> passed pname given the target and attachment.
Updated